exercises/simple-cipher/HINT.md

```diff
@@ -0,0 +1,9 @@
+This exercise has three test classes: StepOne.java, StepTwo.java and StepThree.java. To run all test classes:
```

Comment:
Practitioners might find it confusing to refer to the tests by filenames that doesn't match the actual files.